mmv1/third_party/terraform/utils/config.go.erb

```diff
@@ -194,6 +194,41 @@ func (c *Config) LoadAndValidate(ctx context.Context) error {
 	c.clientEventarcDCL = eventarcDcl.NewClient(c.dclConfig)
 <% end -%>
 
+	// Print Identities executing terraform API Calls.
+
+	if c.ImpersonateServiceAccount != "" {
```

Comment:
Can we wrap this block in a function? That should help tie it together conceptually and we can make sure it doesn't return an error to ensure that this logging is entirely best-effort.

Reply:
Done